proto/swagger/cluster.swagger.json

```diff
@@ -310,75 +319,94 @@
         "createdAt": {
           "type": "string",
           "format": "date-time",
-          "description": "Output. The time that the cluster created."
+          "description": "Output. The time that the cluster created.",
+          "readOnly": true
```

Comment:
@blublinsky the "OUTPUT_ONLY" annotatio has an effect on the generated swagger doc, and it also removes the field from any generated examples. They should be kept.

Reply:
ok